sos/report/plugins/pulp.py

```diff
@@ -72,6 +72,7 @@ def setup(self):
         self.add_copy_spec([
             "/etc/pulp/*.conf",
             "/etc/pulp/settings.py",
+            "/etc/pulp/settings.local.py",
```

Comment:
Do we need to add this to the `postproc()` handling of the "regular" `settings.py` file?

https://github.com/sosreport/sos/blob/f580ee0719b298d3358fc628a7baa690373a5b31/sos/report/plugins/pulp.py#L182

Reply:
By default, customer should not include passwords. But will not warm redact it, in case of any sensitive info.